pegasus/sites.v3/hourofcode.com/public/faq.haml

```diff
@@ -1,11 +1,12 @@
 ---
+title: FAQ | Hour of Code
 layout: wide
 ---
 
-%link{rel: "stylesheet", type: "text/css", href: "/css/faq.css"}
+%link{href: '/css/full-width-section.css', rel:'stylesheet', type:'text/css'}
+%link{rel: "stylesheet", type: "text/css", href: "/css/generated/faq.css"}
 
-:ruby
-  @header["title"] = hoc_s(:hoc_faq_title)
```

Comment:
Curious why we're no longer translating the title?

Reply:
The titles aren't consistent across all of the pages (a lot of others aren't translated and have `… | Hour of Code` at the end) so I wasn't sure what to do here. I can add the translated version here and will make a ticket to update the other titles to be translated.